python/test/unit/test_knobs.py

```diff
@@ -200,7 +194,7 @@ def test_set_knob_directly(fresh_knobs, monkeypatch):
     # Just in case, lets check all the other datatypes too
     fresh_knobs.language.default_fp_fusion = False
     fresh_knobs.amd.use_block_pingpong = True
-    fresh_knobs.amd.global_prefetch = 5
+    fresh_knobs.redis.port = 6380
```

Comment:
What's this `redis.port` change? Can you double check your changes in this file?

Reply:
This part tries to set knobs of different data types. After removing global_prefetch and local_prefetch, there's no integer knob anymore. So I need to find one from other set.

Reply:
Okay yeah sure then. Having a redis port inside Triton would be another level of the game. ;-P